packages/components/src/components/Dropzone/Dropzone.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import type { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { FileWithPath } from "react-dropzone";
+import { Dropzone } from "./Dropzone";
+
+const resolveAfter2Seconds = () => {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve("Document uploaded successfully");
+    }, 2000);
+  });
+};
+
+const sendDocument = async (
+  file: FileWithPath,
+  onProgress?: (progressPercentage: number) => void
+) => {
+  let current = 0;
+  let interval = null;
+
+  const useCallback = () => {
+    if (onProgress !== undefined) {
+      onProgress(current);
+    }
+    const nextCount = current++;
+    if (nextCount === 100) {
+      clearInterval(interval);
+    }
+  };
+
+  interval = setInterval(useCallback, 10);
+
+  if (current < 100) {
+    return await resolveAfter2Seconds();
+  }
+};
+
+const cancelDocumentUpload = (name: string) => {
+  return new Promise(function (resolve, reject) {
+    setTimeout(() => {
+      return reject(new Error(`${name} failed to upload`));
+    }, 1000);
+  });
+};
+
+export default {
+  component: Dropzone,
+  title: "Components/Dropzone",
+} as ComponentMeta<typeof Dropzone>;
+
+const Template: ComponentStory<typeof Dropzone> = (args) => (
+  <Dropzone
+    {...args}
+    cancelDocumentUpload={cancelDocumentUpload}
```

Comment:
Ir our case, we will use dropzone to upload documents, but don't forget that pluto is an open-source project, and people could use it to upload pictures, for example. So, I would suggest creating a prop `cancelUpload` which receives a boolean, and another prop, `onCancelUpload` to have it as a callback.